### PR TITLE
[11.0] [MIG] [RNM] purchase_request_to_rfq_order_approved to purchase_request_order_approved

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
According with #544 
Minor changes on code.

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 server-tools
+purchase-workflow https://github.com/Eficent/purchase-workflow 11.0-mig-purchase_request
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: remove this and rebase, please.

File: purchase_request_order_approved/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request Order Approved",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "version": "11.0.1.0.0",
+    "website": "http://www.eficent.com/",
```
Comment (nikul-serpentcs) [inline purchase_request_order_approved/__manifest__.py]: @hveficent Updated URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: purchase_request_order_approved/tests/test_purchase_request_order_approved.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        vals = {
+            'request_id': purchase_request1.id,
+            'product_id': self.env.ref('product.product_product_6').id,
+            'product_uom_id': self.env.ref('product.product_uom_unit').id,
+            'product_qty': 2.0,
+        }
+        purchase_request_line1 = self.purchase_request_line.create(vals)
+        vals = {
+            'request_id': purchase_request2.id,
+            'product_id': self.env.ref('product.product_product_6').id,
```
Comment (nikul-serpentcs) [inline purchase_request_order_approved/tests/test_purchase_request_order_approved.py]: `self.env.ref('product.product_product_6')`

@hveficent use as a global variable

File: purchase_request_order_approved/tests/test_purchase_request_order_approved.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            'product_uom_id': self.env.ref('product.product_uom_unit').id,
```
Comment (nikul-serpentcs) [inline purchase_request_order_approved/tests/test_purchase_request_order_approved.py]: same here `self.env.ref('product.product_uom_unit')`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
@nikul-serpentcs could you update your review?

**Review by nikul-serpentcs — APPROVED** [review]
LGTM :+1:

**Pull request merged**